src/app.js

```diff
@@ -1,21 +1,48 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+function mv(fileFrom, fileTo) {
+  if (fileFrom === fileTo) {
+    return;
+  }
+
+  const basePath = path.join(__dirname, fileFrom);
+  const content = fs.readFileSync(basePath, 'utf-8');
+
+  try {
+    fs.writeFileSync(fileTo, content);
+    fs.unlinkSync(basePath);
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      throw new Error(e);
+    } else if (e.code === 'EISDIR') {
+      try {
+        const newPath = fileTo[fileTo.length - 1] === '/'
+          ? fileTo + fileFrom
+          : fileTo;
```

Comment:
can we move this check before the first file moving to prevent this error and make write / unlink functionality one time depending on the path?